static/js/util/currency.js

```diff
@@ -1,17 +1,50 @@
 // @flow
 import cc from 'currency-codes';
 import R from 'ramda';
+import iso3166 from 'iso-3166-2';
+
+export const excludedCurrencyCodes = [
```

Comment:
Not currently, but we can build one really quickly (or it may be easier to pass it along with the dashboard API).  @noisecapella what do you think?